Dockerfile.template

```diff
@@ -20,9 +20,6 @@ RUN set -eux \
     ##<version>##
     ##</version>##
     && docker-php-ext-install intl mbstring mysqli bcmath bz2 soap xsl pdo pdo_mysql fileinfo exif zip opcache \
```

Comment:
```suggestion
    && docker-php-ext-install intl mbstring mysqli bcmath gd bz2 soap xsl pdo pdo_mysql fileinfo exif zip opcache \
```

Reply:
happens here anyway: https://github.com/pimcore/docker/blob/master/7.3-Dockerfile-block-1#L1

Reply:
No, that line only configures `gd`.

Reply:
true